app/controllers/users_controller.rb

```diff
@@ -0,0 +1,34 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: [:show, :friends]
+  before_action :set_current_user, only: [:me, :edit]
```

Comment:
1. What is "me"?
2. We can safely refer directly to `current_user` without writing a method where we simply set `@user = current_user`.